src/test/java/uk/gov/ons/ssdc/caseprocessor/testutils/PubsubHelper.java

```diff
@@ -9,11 +9,7 @@
 import com.google.cloud.spring.autoconfigure.pubsub.GcpPubSubProperties;
 import com.google.cloud.spring.pubsub.core.PubSubTemplate;
 import java.io.IOException;
-import java.util.concurrent.ArrayBlockingQueue;
-import java.util.concurrent.BlockingQueue;
-import java.util.concurrent.ExecutionException;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.TimeoutException;
+import java.util.concurrent.*;
```

Comment:
I think it's normally recommend/conventional to import the modules individually rather than import them all

Reply:
Good spot. That'll be because it was auto importing them all